src/EditorFeatures/Core/Implementation/NavigateTo/AbstractNavigateToItemDisplay.cs

```diff
@@ -130,5 +132,9 @@ public void PreviewItem()
             previewService.PreviewItem(this);
         }
 
+        public IReadOnlyList<Span> GetNameMatchRuns(string searchValue)
```

Comment:
this is a just a move.  it makes testing easier as tests don't need to reference dev15 specific binaries.